README.md

```diff
@@ -1,89 +1,51 @@
-# Vet-Clinic-Database
- 
+# Vet_clinic database
+
+is a relational database used to create the database for a vet clinic. I created a table to store animals' information, insert some data into it, and query it.
+
 ## Getting Started
 
 This repository includes files with plain SQL that can be used to recreate a database:
 
 - Use [schema.sql](./schema.sql) to create all tables.
 - Use [data.sql](./data.sql) to populate tables with sample data.
-- Check [queries.sql](./queries.sql) for examples of queries that can be run on a newly created database. **Important note: this file might include queries that make changes in the database (e.g., remove records). Use them responsibly!**
+- Check [queries.sql](./queries.sql) for examples of queries that can be run on a newly created database.
 
```

Comment:
- [ ] Kindly remove this Getting Started section because you already have it below ⬇️ in your readme 🤔